registration/registerTasks.go

```diff
@@ -121,8 +123,10 @@ func init() {
 	baseAgent.RegisterWith(Register)
 	nodeRequirements.RegisterWith(Register)
 	rubyRequirements.RegisterWith(Register)
-	k8sInfra.RegisterWith(Register)
 	k8sEnv.RegisterWith(Register)
+	k8sResources.RegisterWith(Register)
+	k8slogs.RegisterWith(Register)
+	K8sHelm.RegisterWith(Register)
```

Comment:
now that all of them are registered how are you calling them?

Reply:
I've added this 2 new suites:

<img width="1268" alt="image" src="https://github.com/user-attachments/assets/1a6a8abb-eaed-4e0a-82d6-c2c554bb50c2" />

Reply:
If you execute :
nrdiag -h suites

You can see the different suites.

So you can call:
- ` nrdiag -y` --> All
- ` nrdiag -suites K8s `--> Only k8s
-  `nrdiag -suites K8s-agent-control` --> All K8s agent-control